src/types.ts

```diff
@@ -114,6 +114,8 @@ const handleResult = <Input, Output>(
   }
 };
 
+const allowedChecks: ZodStringCheck["kind"][] = ["min", "max", "length", "email", "url", "uuid", "includes", "startsWith", "endsWith", "regex", "trim", "toLowerCase", "toUpperCase", "datetime", "date", "time", "ip", "base64"];
```

Comment:
since we use this a lot now, might be worth the micro-optimization of using a Set